Test/GetMinuteQuarterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once "./src/MyClass.php";
```

Comment:
We discussed before, you can use autoloader from vendor folder

Reply:
I will correct